src/components/rocket/RocketsBookingButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { bookingRocket, bookingCancel } from '../../redux/rockets/rocketsSlice';
+
+const RocketsBookingButton = (props) => {
+  const { id, reserved } = props;
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      {
+        reserved
+          ? <button id={id} type="button" className="btn btn-light" onClick={() => dispatch(bookingCancel(id))}>Cancel Reservation</button>
+          : <button type="button" className="btn btn-primary" onClick={() => dispatch(bookingRocket(id))}>Reserve Rocket</button>
```

Comment:
Kindly store the method to dispatch the action when the button is clicked as a value by declaring a variable above the return keyword. This makes the code cleaner and makes it easy to maintain the code.